discovery-infra/test_infra/helper_classes/nodes.py

```diff
@@ -3,7 +3,10 @@
 import random
 from typing import Dict, Iterator, List
 
+import waiting
+from logger import log
```

Comment:
shouldn't this import be on the first group? (as it's built-in)

Reply:
It's passes the linter(and isortk) . Who am I to question it

Reply:
actually the answer is that it belong to the third group. the linter is probably not able to understand that